### ISSUE TITLE
Large `train/policy_loss` for DiffusionNFT

### ISSUE THREAD

**Issue by lizzy8587** [X-GenGroup/Flow-Factory#48]
Thank you for your great job. I try to train DiffusionNFT on Z-Image using this default [config](https://github.com/X-GenGroup/Flow-Factory/blob/main/examples/nft/full/z_image.yaml) but found the training loss is very large (around 60). Is this expected or any potential bugs on my environment? Thanks for any suggestions!

<img width="3402" height="286" alt="Image" src="https://github.com/user-attachments/assets/023722ca-4a97-46d5-845f-aed37a019d36" />

**Comment by Jayce-Ping** [comment]
I don't know. The loss in the issue you linked seems too small. It is relevant to the reward model as well. As they said,  "The training curve is normal as the reward can increase."
